src/components/InputField/InputField.styled.js

```diff
@@ -59,3 +59,23 @@ export const PasswordReapetLable = styled.p`
   padding-left: 14px;
   color: #49454f;
 `;
+
+export const EyeIcon = styled.div`
```

Comment:
It would be better to give it a name according to its purpose. In fact it isn't an icon, but an icon container.

Reply:
Done.